docs/core/testing/unit-testing-platform-vs-vstest.md

```diff
@@ -52,3 +52,79 @@ To disable the use of the new protocol in Test Explorer, you can edit your proje
+
+### `dotnet test`
+
+Command line options of `dotnet test` are divided into 2 categories: build related arguments and test related ones.
+
+The build related arguments are passed to the `dotnet build` command and as such don't need to be updated for the new platform. Build related arguments are listed below:
+
+- `-a|--arch <ARCHITECTURE>`
+- `--artifacts-path <ARTIFACTS_DIR>`
+- `-c|--configuration <CONFIGURATION>`
+- `-f|--framework <FRAMEWORK>`
+- `-e|--environment <NAME="VALUE">`
+- `--interactive`
+- `--no-build`
+- `--nologo`
+- `--no-restore`
+- `-o|--output <OUTPUT_DIRECTORY>`
+- `--os <OS>`
+- `-r|--runtime <RUNTIME_IDENTIFIER>`
+- `-v|--verbosity <LEVEL>`
+
+The test related arguments are VSTest specific and so need to be transformed to match the new platform. The following table shows the mapping between the VSTest arguments and the new platform:
+
+| VSTest argument | New platform argument |
+|-----------------|-----------------------|
+| `--test-adapter-path <ADAPTER_PATH>` | Not supported |
+| `--blame` | Not supported |
+| `--blame-crash` | `--crashdump` requires [Crash dump extension](./unit-testing-platform-extensions-diagnostics.md#crash-dump) |
+| `--blame-crash-dump-type <DUMP_TYPE>` | `--crashdump-type` requires [Crash dump extension](./unit-testing-platform-extensions-diagnostics.md#crash-dump) |
+| `--blame-crash-collect-always` | Not supported |
+| `--blame-hang` | `--hangdump` requires [Hang dump extension](./unit-testing-platform-extensions-diagnostics.md#hang-dump) |
+| `--blame-hang-dump-type <DUMP_TYPE>` | `--hangdump-type` requires [Hang dump extension](./unit-testing-platform-extensions-diagnostics.md#hang-dump) |
+| `--blame-hang-timeout <TIMESPAN>` | `--hangdump-timeout` requires [Hang dump extension](./unit-testing-platform-extensions-diagnostics.md#hang-dump) |
+| `--collect <DATA_COLLECTOR_NAME>` | Depends on the data collector |
+| `-d\|--diag <LOG_FILE>` | `--diagnostic` |
+| `--filter <EXPRESSION>` | Depends upon the selected test framework |
+| `-l\|--logger <LOGGER>` | Depends on the logger |
+| `--results-directory <RESULTS_DIR>` | `--results-directory <RESULTS_DIR>` |
+| `-s\|--settings <SETTINGS_FILE>` | Depends upon the selected test framework |
+| `-t\|--list-tests` | `--list-tests` |
+| `-- <RunSettings arguments>` | Not supported |
+
+> [!IMPORTANT]
+> Before specifying any `Microsoft.Testing.Platform`, you need to add `--` to separate the `dotnet test` arguments from the new platform arguments.
+
+### `vstest.console.exe`
+
+If you are using `vstest.console.exe` directly, we recommend replacing it with the `dotnet test` command.
+
+### Test Explorer
+
+When using Visual Studio or Visual Studio Code Test Explorer, you might need to enable the support for the new test platform.
+
+#### Visual Studio
+
+Visual Studio Test Explorer supports the new test platform starting with version 17.14. If you are using an earlier version, you might need to update your Visual Studio to the latest version.
```

Comment:
This seems very misleading as it should be working in 17.13 for example (likely working in even older versions).

@drognanar Can you share a bit of info here please so we can improve the documentation? This statement causes readers to think they can't migrate until 17.14 is released stable (it's currently preview)